### DISCUSSION TITLE
Unable to write a commit message

### DISCUSSION THREAD

**Discussion by zer09** [NeogitOrg/neogit#1453] [Q&A] [answered]
I setup a new system with `manjaro` then copy my config to the new system but when I try to write a commit message I only see this(see picture below) and unable to write the commit message.

![Screenshot 2024-08-23 104145](https://github.com/user-attachments/assets/ad962c70-a387-4a0f-ab1e-1fe9d918d695)

It has no more details other than that.

Any advice what to do, or what to look to get more info, to fix this problem.

neovim version:
NVIM v0.10.1
Build type: Release
LuaJIT 2.1.1720049189

**Comment by zer09** [comment, marked as answer]
I was forgot the setup the config for the email and username

**Comment by DrTeagle** [comment]
can you be more descriptive pls, having a similair issue

**Reply by zer09** [reply]
Setup your email and username in your git config

**Comment by DrTeagle** [comment]
I have that, running this command in teh specific repo that doesnt work causes it to work and I have no idea why?:
```bash
git config core.hooksPath .git/hooks
```

**Reply by zer09** [reply]
Try manual run git commit on the terminal

**Reply by DrTeagle** [reply]
that works, its always worked its just in neogit that it doesnt work
